### PR TITLE
VIH-5431 Added unit test for Judge events to event factory

### PR DESCRIPTION
**Pull request by valentinabennett**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-5431

### Change description ###

Added missing test for judge statuses

**Does this PR introduce a breaking change?** (check one with "x")

```
[X ] Yes
[ ] No
```

### CODE REVIEW

File: VideoAPI/VideoApi.AcceptanceTests/Features/Events.feature [hunk 1]
```diff
@@ -5,7 +5,25 @@
 
 Scenario: Add conference event
 	Given I have a conference
-    And I have a valid conference event request for event type Joined
-    When I send the request to the endpoint
-    Then the response should have the status NoContent and success status True
+	And I have a valid conference event request for event type Joined
```
Comment (shaed-parkar) [inline VideoAPI/VideoApi.AcceptanceTests/Features/Events.feature]: please stop checking in white spaces

File: VideoAPI/VideoApi.AcceptanceTests/Features/Events.feature [hunk 2]
```diff
@@ -5,7 +5,25 @@
...
+
+	Examples:
+		| EventType             |
+		| Joined                |
+		| Disconnected          |
+		| Pause                 |
+		| Close                 |
+		| Leave                 |
+		| JudgeAvailable        |
+		| JudgeUnavailable      |
```
Comment (shaed-parkar) [inline VideoAPI/VideoApi.AcceptanceTests/Features/Events.feature]: this line should be the only change

File: VideoAPI/VideoApi.AcceptanceTests/Features/Events.feature (hunk 1 above)
Comment (valentinabennett) [reply VideoAPI/VideoApi.AcceptanceTests/Features/Events.feature]: I don't know that do you mean, have we any standards for formatting?

File: VideoAPI/VideoApi.AcceptanceTests/Features/Events.feature (hunk 2 above)
Comment (valentinabennett) [reply VideoAPI/VideoApi.AcceptanceTests/Features/Events.feature]: Need to introduce standard, in my case VS2019 use default to format it

Comment (valentinabennett) [reply VideoAPI/VideoApi.AcceptanceTests/Features/Events.feature]: Sorry Shaed, but I could not remove these white space now, it's difficult as I dont't know how it was before and VS does not show it. I think it's not so important at the moment. But next time I will adjust the indent and tab in my VS to be agreed with the standard we set.

**Pull request merged**